src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/x64/conv_concat.cpp

```diff
@@ -159,7 +156,6 @@ namespace ConvolutionConcat {
 /* ============= Convolution (2D) ============= */
 const std::vector<CPUSpecificParams> CPUParams2D = {
     conv_ref_2D,
```

Comment:
What about ref? Don't see big value in explicit testing on out side (and given ref conv is tremendously slow it might noticable affect scope timings).

Reply:
let's remove it as well